components/ADempiere/DataTable/Windows/index.vue

```diff
@@ -409,6 +409,19 @@ export default defineComponent({
       }
     }
 
+    /**
+     * isDisplayedDefaultTable
+     */
+    function isDisplayedDefaultTable({ isMandatory, isParent, defaultValue, displayType, parsedDefaultValue, name }) {
+      if (displayType === 15) {
```

Comment:
Please import constant on src/utils/ADempiere/references.js and compare with CONSTANT.id